lib/mocks/objects.ts

```diff
@@ -55,7 +55,7 @@ export const employer: types.Employer = {
 
 export const position: types.Position = {
   title: 'Midfielder',
-  start_date: '2006-01-01',
+  hiring_date: '2006-01-01',
```

Comment:
I believe we want `hired_date` here:

![image](https://user-images.githubusercontent.com/60897327/100135238-34959380-2e3e-11eb-958f-3519f2f6eb2f.png)

Reply:
😱